static/katex.less

```diff
@@ -49,6 +49,18 @@
         display: inline-block;
     }
 
+    .mathrm {
+        font-style: normal;
+    }
+
+    .textit {
+        font-style: italic;
+    }
+
+    .textsc {
+        font-variant: small-caps;
+    }
```

Comment:
I was looking at http://caniuse.com/#feat=font-variant-alternates which appears to be some extension to `font-variant`.